lib/tasks/decidim_app.rake

```diff
@@ -48,8 +48,6 @@ namespace :decidim_app do
       Rake::Task["decidim:repair:url_in_content"].invoke
       puts "Running decidim:repair:translations"
       Rake::Task["decidim:repair:translations"].invoke
```

Comment:
Please revert this remove because k8s operator must run migrations on upgrade

Reply:
I agree, it seemed to be working pretty well, but now it crashes the test and I don't really know why so I took back the version of the decidim-app

Reply:
After having a look to the rake task, migration rake task is duplicated so we should remove it as it !